_posts/2024-01-19-pyhc-summer-school-2024-announced.markdown

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: "A Much-anticipated Sequal: the PyHC 2024 Summer School"
```

Comment:
I wasn't sure about this capitalization so I checked, and in standard title case the "Anticipated" would be capitalized in "Much-Anticipated". Can you please capitalize it?

Reply:
valid; I wondered that myself.